assignments/prototypes.js

```diff
@@ -15,13 +15,33 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function GameObject(atti){
+  Humanoid.call(this, atti);
+}
+
```

Comment:
Your inheritance is backwards here. This is actually the Parent or base constructor and shouldn't be calling or needing to bind the `this` keyword. This should have just been:
```js
function GameObject(attributes) {
	this.createdAt = attributes.createdAt;
	this.dimensions = attributes.dimensions;
}
```